packages/ckeditor5-utils/tests/env.js

```diff
@@ -326,6 +351,44 @@ describe( 'Env', () => {
 
 			expect( isMediaForcedColors() ).to.be.false;
 		} );
+
+		it( 'returns false if window object is not available', () => {
+			// `global.window` is an empty object if `window` was not available in global space.
+			const _window = global.window;
+			global.window = {};
+
+			expect( isMediaForcedColors() ).to.be.false;
+
+			global.window = _window;
+		} );
+	} );
+
+	describe( 'isMotionReduced()', () => {
```

Comment:
It is already tested above

Reply:
The above test checks `env.isMotionReduced`. I added additional tests for the util to match the other getter. I guess no harm in them. Also, I added `window = {}` test here.